src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import { representativeFields } from 'applications/caregivers/definitions/constants';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+// const { representative } = fullSchema.properties;
```

Comment:
Leaving this in here as a reminder to integrate schema but will be merging as functional under a feature flip for testing